packages/validator/src/lib/psi/PsiResult.ts

```diff
@@ -30,17 +33,18 @@ export type LighthouseCategories = {
   [key in LighthouseCategoryName]: LighthouseCategory;
 };
 
-export type LighthouseAuditDetailsItem = {
-  [key: string]: number;
+export type LighthouseMetricAudit = {
```

Comment:
Is there some technical reason this can't just be:

```
export type LighthouseMetricAudit = {
  firstContentfulPaint: number;
  largestContentfulPaint: number;
  maxPotentialFID: number;
  cumulativeLayoutShift: number:
}
```

Reply:
(also, if we were to go this route, I'm wondering whether we want to type the numbers - eg "percent", "milliseconds" or something)

Reply:
Oh, gotcha - I thought the concern was the unbounded string (this approach does give us nice autocomplete).

The reason I went this route was two things: 1 - there are dozens of keys on this object on the JSON. 2 - The same keys are used in other places in the JSON, and this would allow us to re-use if we need in the future.

I'll move to your suggestion for now and we can review in the future if we need to change.

Reply:
done